### ISSUE TITLE
False Negative: SF_SWITCH_NO_DEFAULT missing when the last case contains only a break statement

### ISSUE THREAD

**Issue by leemeii** [spotbugs/spotbugs#3905]
**Describe the bug**
The rule **SF_SWITCH_NO_DEFAULT** fails to report a missing default case in a switch statement if the last case block is "empty" (contains only a break statement). However , if the last case block contains any logic , the bug is correctly identified. 
In both scenarios, the default case is missing, and both should be flagged according to the rule definition.

**To Reproduce**
```java
package test;

public class LastCaseContentTest {
    
   
    public void test1() {
        int x = 0;
        switch(x) {
            case 0: x = 0; break;
            case 1: x = 1; break; // <- True Positive
    }

    public void test2() {
        int x = 0;
        switch(x) {
            case 0: x = 0; break;
            case 1: break; // <- should report SF (False Negative)
        }
    }
}
```

**Expected behavior**
SpotBugs should report SF_SWITCH_NO_DEFAULT for both test1() and test2(). The presence or absence of a default case is independent of the content of the preceding case blocks.

**Actual behavior**
Only test1() is reported. test2() is silently ignored .

**Environment**
- Java version: JDK 17
- SpotBugs Version: 4.9.8
- How do you use SpotBugs: CLI
